test/e2e/network/netpol/model.go

```diff
@@ -21,11 +21,28 @@ import (
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
+const (
+	CPURequest    = "1m"
```

Comment:
Where did these constants come from?  Aren't the containers just running agnhost or something?

I'm surprised they'd need this much CPU and memory, can we collect some data showing how much they use during a test run?

Reply:
How much smaller can you get then 1m?

Reply:
for the analysis of the data, https://docs.google.com/spreadsheets/d/1a9ltOSV8AaGlKA1CZM_VW_D9lECc4W1x8hiyKuyLfoA/edit?usp=sharing

Reply:
I'm asking about memory request, cpu limit, memory limit 😄 -- does agnhost actually need that much?  I don't remember.

Is it possible to include a boiled-down summary of that data from the google spreadsheet as a comment?  Otherwise these just look like arbitrary constants with no context, and no way to tell if they're right or wrong.

Reply:
+1 to include summary  as comments, and reference this issue (not as a proof that these numbers are perfect, but just as a "these empirically work and fence the total cpu usage")